### ISSUE TITLE
Pipeline Owner set to None

### ISSUE THREAD

**Issue by sarodspirit** [pmbanugo/heroku-review-app-actions#7]
This is somehow forcing us to change the pipeline ownership to NONE, as it is not deploying a new review app but attempting to create an app.

```
{"id":"conflict","message":"The app and pipeline do not have the same owner. Consider transferring the app to the pipeline's owner before trying again. Pipelines can only contain mixed ownership apps when the pipeline owner is set to 'none'."}
```

**Comment by pmbanugo** [comment]
There's not much difference between an app and a review app. They're practically the same. I think you're getting this error because the `api-key` (your Heroku API Key) belongs to someone that's not the owner of the pipeline. Try changing using the API key of the person who owns or created the pipeline. 

Let me know if that works

**Comment by jumichot** [comment]
You need to create a team app (not a normal app) if the owner of the pipeline is a team

**Comment by guioconnor** [comment]
I may be missing something but v1.0.0 does not run into this problem.

**Comment by cderche** [comment]
> You need to create a team app (not a normal app) if the owner of the pipeline is a team

I'm guessing that isn't currently available in this action.

**Comment by jumichot** [comment]
> I'm guessing that isn't currently available in this action.
No it's not, I had to edit this action and change the endpoints for creating the app : ` curl -n -X POST https://api.heroku.com/teams/apps`

**Comment by pmbanugo** [comment]
See my comment [here](https://github.com/pmbanugo/heroku-review-app-actions/issues/9#issuecomment-1117881281) for an update. It only takes into account if the repo has app.json (for now), and will expand on it later.

**Comment by pmbanugo** [comment]
Fixed in Releases [v1.2.1](https://github.com/pmbanugo/heroku-review-app-actions/releases/tag/v1.2.1) and [v2](https://github.com/pmbanugo/heroku-review-app-actions/releases/tag/v2.0.0)
